src/styles/index.scss

```diff
@@ -1,3 +1,12 @@
+@use './blocks/calendar.scss';
+@use './utils/variables.scss';
+
 body {
-  margin: 0;
+  margin: 0 auto;
+  padding: 10px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  width: 1000px;
```

Comment:
The width of the body is set to a fixed value of 1000px. This could cause layout issues on smaller screens. Consider using relative units or media queries to ensure the layout is responsive.